demos/demo-async-rest/demo-async-rest-webapp/src/main/webapp/index.html

```diff
@@ -1,44 +1,62 @@
 <html>
  <head>
+  <link rel="stylesheet" href="demo.css"/>
   <style type='text/css'>
     iframe {border: 0px}
     table, tr, td {border: 0px}
   </style>
 </head>
 <body>
-<h1>Blocking vs Asynchronous REST</h1>
-<p>
-This demo calls the EBay WS API both synchronously and asynchronously,
-to obtain items matching each of the keywords passed on the query
-string.  The time the request thread is held by the servlet is displayed in red for both.
-</p>
+    <div class="topnav">
+      <a class="menu" href="http://localhost:8080/">Demo Home</a>
+      <a class="menu" href="https://github.com/eclipse/jetty.project/tree/jetty-10.0.x/demos/demo-async-rest">Source</a>
```

Comment:
This will need to be manually changed for the merge to 11.0.x

Reply:
Yes. Only the once though.